Formula/kubekey.rb

```diff
@@ -0,0 +1,33 @@
+class Kubekey < Formula
+  desc "Installer for Kubernetes and / or KubeSphere, and related cloud-native add-ons"
+  homepage "https://kubesphere.io"
+  url "https://github.com/kubesphere/kubekey.git",
+      tag:      "v1.2.1",
+      revision: "94aa580f7ae60792044eddfa3b3e38b8a109d79e"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on :linux # some commands fail to run successfully on macOS
```

Comment:
If it compiles on macOS we should probably just allow it to build there.

Reply:
It compiles, but one of the main commands, `kk create cluster` will fail to execute on macOS because it shells out to something like `tar cfz - -C %s -T /dev/stdin <<< network-plugin.yaml | base64 --wrap=0`, which will fail on macOS due to options incompatibilities. But the issue should be resolved in the next version 2.0.0.
What do you think? Should I remove the Linux dependency or leave it in with a comment to remove on v2.0.0 release?

Reply:
@SMillerDev Thoughts on the above? 😃

Reply:
If only one of the command fails on macOS I think we should just allow it to build on macOS

Reply:
Okay, I removed the line